tests/integration_tests/http_api/util/test_case.py

```
@@ -74,6 +77,49 @@ def create_changefeed(sink_uri):
+                    {
+                        "matcher": ["*.*"],
+                        "partition": "columns",
+                        "columns": ["a.b"],
+                        "topic": "persistent://public/default/http_api_{schema}_{table}",
+                    }
+                ]
+            }
+        }
+    })
+    headers = {"Content-Type": "application/json"}
+    resp = rq.post(url, data=data, headers=headers)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)